### PR TITLE
W-12481950-variable-limitation-cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc [hunk 1]
```diff
@@ -10,6 +10,22 @@ image:toolbox-activity-parameters.png[The Activity Parameters Action Step, 40%,
 
 Only one _Activity Parameters_ Action Step can exist in a process and it is automatically placed in the _Workflow Initialization_ section when creating a new Workflow.
 
+Activity Parameters consist of a name, a description, and a value. Activity parameter names have the following requirements:
```
Comment (dhirsch1) [inline rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc:13]:
```suggestion
Activity Parameters consist of a name, description, and value. Activity parameter names have the following requirements:

```

File: rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc [hunk 2]
```diff
@@ -10,6 +10,22 @@ image:toolbox-activity-parameters.png[The Activity Parameters Action Step, 40%,
...
+
+* The name consists of a combination of letters, numbers, and underscores.
+* The name contains no special characters or spaces.
+* The initial letter of the name is lower case.
+* The name is unique.
++
+You cannot use the same name multiple times for different variables.
+
+The predefined values are test values that RPA Builder uses only during a local run. The variables register automatically on RPA Manager when uploaded. RPA Manager enables you to modify the values of the variables and define them differently, depending on the station on which the Workflow is being executed. See xref:advanced-concepts-using-variables.adoc[Using Variables] for additional information.
```
Comment (dhirsch1) [inline rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc:22]:
```suggestion
The predefined values are test values that RPA Builder uses only during a local run. The variables register automatically on RPA Manager when uploaded. RPA Manager enables you to modify the values of the variables and define them differently, depending on the station on which the Workflow is executed. See xref:advanced-concepts-using-variables.adoc[Using Variables] for additional information.

```

File: rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc [hunk 3]
```diff
@@ -28,6 +44,8 @@ image:add-new-parameters.png[The list of available activity parameters to add, 3
 The following data types are available:
 +
 * Texts (*alphanumeric*)
++
+The value for text(alphanumeric) variables has a limit of 4000 characters.
```
Comment (dhirsch1) [inline rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc:48]:
```suggestion
The value for text (alphanumeric) variables has a limit of 4000 characters. 

```

File: rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc [hunk 4]
```diff
@@ -37,25 +55,9 @@ The following data types are available:
...
-The predefined values are test values that are only used for a local run
-within RPA Builder. The variables are registered automatically
-on RPA Manager when uploaded. There, you can modify the values of the
-variables and define them differently, depending on the station on which
-the Workflow is being executed. See xref:advanced-concepts-using-variables.adoc[Using Variables] for additional information.
-
-Please note that the test value may change in the course of the BP due
-to the possible assignment of new values. If the BP is started again,
-the value will be reset to its initial value.
+. Define the *Name*, *Description*, and *Variable Value* for the Activity Parameter.
```
Comment (dhirsch1) [inline rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc:58]: Does "Parameter" have to be initial capped? Usually, it is lowercase.

File: rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc [hunk 5]
```diff
@@ -10,6 +10,22 @@ image:toolbox-activity-parameters.png[The Activity Parameters Action Step, 40%,
...
+* The name consists of a combination of letters, numbers, and underscores.
+* The name contains no special characters or spaces.
+* The initial letter of the name is lower case.
+* The name is unique.
++
+You cannot use the same name multiple times for different variables.
+
+The predefined values are test values that RPA Builder uses only during a local run. The variables register automatically on RPA Manager when uploaded. RPA Manager enables you to modify the values of the variables and define them differently, depending on the station on which the Workflow is being executed. See xref:advanced-concepts-using-variables.adoc[Using Variables] for additional information.
+
+Please note that the test values may change in the course of the business process due
```
Comment (dhirsch1) [inline rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc:24]: The test values might change in the course of the business process due

```

Comment (dhirsch1) [reply rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc:24]: I would get rid of "Please note that." if you want to keep it, my suggestion is to remove "Please."

**Review by dhirsch1 — APPROVED** [review]

**Review by dhirsch1 — APPROVED** [review]

**Review by justus-schach — COMMENTED** [review]
For now the limitation of 4000 characters should only apply to Alphanumeric values.

**Pull request merged**